src/main/java/org/mskcc/limsrest/service/GetReadyForIllumina.java

```diff
@@ -266,9 +272,12 @@ private RunSummary createRunSummaryForNonPooledSamples(DataRecord unpooledSample
         summary.setRecipe(getRecipeForSample(unpooledSample, user));
         summary.setPlateId((String) sampleFieldValues.getOrDefault("RelatedRecord23", ""));
         String indexAndBarcode = getSampleLibraryIndexIdAndBarcode(unpooledSample, user);
```

Comment:
Method signature is a bit non-standard returning "index,barcode" then splitting it but I guess we can leave for now.